*** ./smali/android/content/res/Resources.smali	2016-11-14 15:11:10.484963400 +0800
--- /media/finder/Seagate/patchrom/shamu_new/temp/new_smali/framework.jar.out/./smali/android/content/res/Resources.smali	2016-11-14 15:05:49.494220000 +0800
***************
*** 215,221 ****
      return-void
  .end method
  
! .method private constructor <init>()V
      .locals 4
  
      .prologue
--- 215,221 ----
      return-void
  .end method
  
! .method constructor <init>()V
      .locals 4
  
      .prologue
***************
*** 851,860 ****
      .local v0, "ret":Landroid/content/res/Resources;
      if-nez v0, :cond_0
  
!     new-instance v0, Landroid/content/res/Resources;
  
      .end local v0    # "ret":Landroid/content/res/Resources;
!     invoke-direct {v0}, Landroid/content/res/Resources;-><init>()V
  
      .restart local v0    # "ret":Landroid/content/res/Resources;
      sput-object v0, Landroid/content/res/Resources;->mSystem:Landroid/content/res/Resources;
--- 851,860 ----
      .local v0, "ret":Landroid/content/res/Resources;
      if-nez v0, :cond_0
  
!     new-instance v0, Landroid/content/res/MiuiResources;
  
      .end local v0    # "ret":Landroid/content/res/Resources;
!     invoke-direct {v0}, Landroid/content/res/MiuiResources;-><init>()V
  
      .restart local v0    # "ret":Landroid/content/res/Resources;
      sput-object v0, Landroid/content/res/Resources;->mSystem:Landroid/content/res/Resources;
***************
*** 4527,4550 ****
      .end local v12    # "dr":Landroid/graphics/drawable/Drawable;
      .end local v14    # "cs":Landroid/graphics/drawable/Drawable$ConstantState;
      :cond_4
-     sget-object v4, Landroid/content/res/Resources;->sPreloadedDrawables:[Landroid/util/LongSparseArray;
- 
      move-object/from16 v0, p0
  
!     iget-object v5, v0, Landroid/content/res/Resources;->mConfiguration:Landroid/content/res/Configuration;
! 
!     invoke-virtual {v5}, Landroid/content/res/Configuration;->getLayoutDirection()I
! 
!     move-result v5
! 
!     aget-object v4, v4, v5
  
!     invoke-virtual {v4, v10, v11}, Landroid/util/LongSparseArray;->get(J)Ljava/lang/Object;
  
      move-result-object v14
  
-     check-cast v14, Landroid/graphics/drawable/Drawable$ConstantState;
- 
      .restart local v14    # "cs":Landroid/graphics/drawable/Drawable$ConstantState;
      goto :goto_2
  
--- 4527,4540 ----
      .end local v12    # "dr":Landroid/graphics/drawable/Drawable;
      .end local v14    # "cs":Landroid/graphics/drawable/Drawable$ConstantState;
      :cond_4
      move-object/from16 v0, p0
  
!     move/from16 v1, p2
  
!     invoke-virtual {v0, v10, v11, v1}, Landroid/content/res/Resources;->getPreloadedDrawable(JI)Landroid/graphics/drawable/Drawable$ConstantState;
  
      move-result-object v14
  
      .restart local v14    # "cs":Landroid/graphics/drawable/Drawable$ConstantState;
      goto :goto_2
  
